packs/data/boons-and-curses.db/achaekek-major-curse.json

```diff
@@ -0,0 +1,38 @@
+{
+    "_id": "GJdHcY56q2c3kSiA",
+    "data": {
+        "badge": null,
+        "description": {
+            "value": "<p><strong>Major Curse:</strong> The Mantis God doesn't waste his time toying with or tormenting those who truly anger him, or those who dare to consider themselves divine. He rips open a portal to your location, kills you, drags your soul to judgment in a way that prevents resurrection magic, and then leaves.</p>"
```

Comment:
Does this even need an effect?

Reply:
Realistically, no, but I added it just for the sake of GM reference and completeness.